src/sentry/native/native_sdk.cpp

```diff
@@ -223,7 +223,7 @@ void NativeSDK::initialize() {
 
 	sentry_options_set_database_path(options, (OS::get_singleton()->get_user_data_dir() + "/sentry").utf8());
 	sentry_options_set_sample_rate(options, SentryOptions::get_singleton()->get_sample_rate());
-	sentry_options_set_release(options, SentryOptions::get_singleton()->get_release());
+	sentry_options_set_release(options, SentryOptions::get_singleton()->get_release().utf8());
 	sentry_options_set_debug(options, SentryOptions::get_singleton()->is_debug_enabled());
 	sentry_options_set_environment(options, sentry::environment::get_environment());
```

Comment:
```suggestion
	sentry_options_set_environment(options, sentry::environment::get_environment()).utf8();
```
Do we need to do that here too?

Reply:
Yeah, as we discussed, this returns a C string handler which we need to pass to `sentry_native` call.
The data type has changed for `release` property to `godot::String`, so this is now necessary, or otherwise it won't compile.